### PR TITLE
LPD-32137 Create upgrade process to standardize the ERC of system definitions

### PR DESCRIPTION
**Pull request by devpedrotavares**
Follow up: https://github.com/brianchandotcom/liferay-portal/pull/152891

### CODE REVIEW

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v9_2_2/ObjectDefinitionUpgradeProcess.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+		try (
+			PreparedStatement preparedStatement1 = connection.prepareStatement(sql);
+
+			PreparedStatement preparedStatement2 =
+				AutoBatchPreparedStatementUtil.concurrentAutoBatch(
+					connection,
+					"update ObjectDefinition set externalReferenceCode = ? " +
+					"where objectDefinitionId = ?");
+
+			ResultSet resultSet = preparedStatement1.executeQuery();) {
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v9_2_2/ObjectDefinitionUpgradeProcess.java:36]: Remove the spaces inside the try-with-resource statement and remove the semicolon from `ResultSet resultSet = preparedStatement1.executeQuery();`

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v9_2_2/ObjectDefinitionUpgradeProcess.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+/**
+ * @author Pedro Tavares
+ */
+public class ObjectDefinitionUpgradeProcess extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		String sql = SQLTransformer.transform(
+			"select name, objectDefinitionId from ObjectDefinition where " +
+				"system_ = [$TRUE$] AND modifiable = [$FALSE$]");
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v9_2_2/ObjectDefinitionUpgradeProcess.java:25]: AND should be and

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by devpedrotavares** [comment]
ci:forward

**Comment by devpedrotavares** [comment]
ci:forward:force

**Pull request closed without merging**